src/analysis/Environments/Environments.ts

```diff
@@ -66,18 +64,30 @@ type LeoAppProviderNeeds = Pick<LeoAppProvider, 'listWithoutProject' | 'get' | '
 type LeoRuntimeProviderNeeds = Pick<LeoRuntimeProvider, 'list' | 'stop' | 'delete'>;
 type LeoDiskProviderNeeds = Pick<LeoDiskProvider, 'list' | 'delete'>;
 
+export interface DataRefreshInfo {
+  leoCallTimeMs: number;
+  totalCallTimeMs: number;
+  runtimes: number;
+  disks: number;
+  apps: number;
+}
+
+export interface EnvironmentsEvents {
+  onDataRefresh: DataRefreshInfo;
```

Comment:
I think the convention is that `onFoo` is the handler for a `foo` event, but the event name does not include "on".

For example, native "click" events vs "onClick".
https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent#javascript

Our current metrics events also do not include "on" in the event names.
https://github.com/DataBiosphere/terra-ui/blob/c85a4fdd16d8fc70a0e94eff603442d20abef859/src/libs/events.ts#L17-L180

```suggestion
  dataRefresh: DataRefreshInfo;
```

Reply:
good call. changing over...